CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ an upcoming release.
 * **Logging**
   * [SECURITY] Deploy OpenSearch Dashboards pod with readOnlyRootFilesystem enabled
 
+* **Tracing**
+  * [FEATURE] Introduced new **experimental** ability to deploy tracing pipeline which includes Tempo and configuring Fluentbit to ingest traces and output them to Tempo. As it is experimental, there is work left to be done, including documentation to accompany this features.
```

Comment:
Fluent Bit (two words) NOT Fluentbit.
And a bit wordy ;-) ...but fine otherwise.